commit 644c41b3f88b961f78c26f414494c2473d552f70 (HEAD -> master)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-09-06T15:35:31-04:00 (4 weeks ago)
    
    MANTA-4520 node-libmanta transitive dtrace-provider depdencies too old to build on x86_64
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index cdd4ab1..6a514e4 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "libmanta",
     "description": "common functions et al for manta services",
-    "version": "1.3.0",
+    "version": "1.4.0",
     "author": "Joyent (joyent.com)",
     "main": "./lib/index.js",
     "repository": {
@@ -11,12 +11,12 @@
     "dependencies": {
         "assert-plus": "0.1.5",
         "backoff": "2.3.0",
-        "bunyan": "0.22.1",
+        "bunyan": "1.8.12",
         "jsprim": "^1.3.1",
         "lru-cache": "2.3.1",
-        "moray": "~3.5.0",
+        "moray": "~3.7.0",
         "once": "1.3.0",
-        "restify": "2.6.3",
+        "restify": "4.3.4",
         "redis": "0.10.1",
         "vasync": "1.6.3",
         "verror": "^1.10.0",
@@ -25,14 +25,14 @@
     "devDependencies": {
         "cmdutil": "0.1.0",
         "extsprintf": "1.3.0",
-        "libuuid": "0.1.2",
+        "libuuid": "0.2.1",
         "lstream": "0.0.4",
-        "manta": "1.5.2",
+        "manta": "5.2.1",
         "nodeunit": "0.9.1",
         "posix-getopt": "1.2.0"
     },
     "engines": {
-        "node": ">=0.8.22"
+        "node": ">=0.10.48"
     },
     "scripts": {
         "test": "./node_modules/.bin/nodeunit test/*.test.js"
